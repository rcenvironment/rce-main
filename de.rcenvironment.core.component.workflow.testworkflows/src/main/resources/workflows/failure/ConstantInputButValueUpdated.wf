{
  "identifier" : "1bddadb7-7a08-4405-8403-626a1944e8fe",
  "workflowVersion" : "4",
  "nodes" : [ {
    "identifier" : "b73e3aa9-7b94-42ff-a17a-7d6bdf26e5b6",
    "name" : "Constant",
    "location" : "235:102",
    "active" : "true",
    "component" : {
      "identifier" : "de.rcenvironment.script",
      "version" : "3.4",
      "name" : "Script"
    },
    "configuration" : {
      "pythonExecutionPath" : "${pythonExecutionPath}",
      "usageOfScript" : "NEW",
      "script" : "print 'Design variable: ' + str( RCE.read_input(\"design_variable\"))\r\nRCE.write_output(\"result\", RCE.read_input(\"design_variable\") * 2)",
      "scriptLanguage" : "Jython"
    },
    "dynamicInputs" : [ {
      "identifier" : "22006f78-2cbe-4baa-880c-b421b97e7dc0",
      "name" : "design_variable",
      "epIdentifier" : "default",
      "datatype" : "Float",
      "metadata" : {
        "inputExecutionConstraint_4aae3eea" : "Required",
        "inputHandling_73b1056e" : "Constant"
      }
    } ],
    "dynamicOutputs" : [ {
      "identifier" : "004b9238-66c1-4b35-8bb7-e59a6199f684",
      "name" : "result",
      "epIdentifier" : "default",
      "datatype" : "Float"
    } ]
  }, {
    "identifier" : "ebe3e9bd-58b0-45ad-bff7-be4df4546da7",
    "name" : "Parametric Study",
    "location" : "86:102",
    "active" : "true",
    "component" : {
      "identifier" : "de.rcenvironment.parametricstudy",
      "name" : "Parametric Study",
      "version" : "3.3"
    },
    "configuration" : {
      "FromValue" : "0.0",
      "StepSize" : "0.5",
      "ToValue" : "21.0"
    },
    "dynamicInputs" : [ {
      "identifier" : "6c413c68-463c-48ab-a7d3-616c8d8771cc",
      "name" : "param",
      "epIdentifier" : "parameters",
      "datatype" : "Float"
    } ],
    "staticOutputs" : [ {
      "identifier" : "6b5ed2db-ba9b-4e37-a4bf-06a73d731b01",
      "name" : "Design variable",
      "datatype" : "Float",
      "metadata" : {
        "FromValue" : "0.0",
        "StepSize" : "0.000001",
        "ToValue" : "10.0"
      }
    } ]
  } ],
  "connections" : [ {
    "source" : "ebe3e9bd-58b0-45ad-bff7-be4df4546da7",
    "output" : "6b5ed2db-ba9b-4e37-a4bf-06a73d731b01",
    "target" : "b73e3aa9-7b94-42ff-a17a-7d6bdf26e5b6",
    "input" : "22006f78-2cbe-4baa-880c-b421b97e7dc0"
  }, {
    "source" : "b73e3aa9-7b94-42ff-a17a-7d6bdf26e5b6",
    "output" : "004b9238-66c1-4b35-8bb7-e59a6199f684",
    "target" : "ebe3e9bd-58b0-45ad-bff7-be4df4546da7",
    "input" : "6c413c68-463c-48ab-a7d3-616c8d8771cc"
  } ]
}