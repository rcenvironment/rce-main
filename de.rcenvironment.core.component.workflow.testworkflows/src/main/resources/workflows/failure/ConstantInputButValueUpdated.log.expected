A second value at input 'design_variable' of type 'constant' received. Only one value is allowed. First: '0.0'